dask/dataframe/utils.py

```diff
@@ -24,9 +24,18 @@
 from . import _dtypes  # noqa: F401 register pandas extension types
 from . import methods
 from ._compat import PANDAS_GT_100, PANDAS_GT_110, PANDAS_GT_120, tm  # noqa: F401
-from .dispatch import make_meta, meta_nonempty  # noqa : F401
+from .dispatch import make_meta  # noqa : F401
+from .dispatch import make_meta_obj, make_meta_util, meta_nonempty  # noqa : F401
 from .extensions import make_scalar
 
+meta_object_types = (pd.Series, pd.DataFrame, pd.Index, pd.MultiIndex)
+try:
+    import scipy.sparse as sp
+
+    meta_object_types += (sp.csr.csr_matrix,)
```

Comment:
Sorry missed this one

```suggestion
    meta_object_types += (sp.spmatrix,)
```